dev_guide/builds.adoc

```diff
@@ -1184,6 +1190,80 @@ spec:
 ----
 ====
 
+[[build-hooks]]
+
+== Build Hooks
+Build hooks can be used as a part of the build flow to run tests inside a temporary container and validate your image before it is being pushed to the registry. The command is executed with the current working directory set to the image's `WORKDIR`.
+The hook will fail if the script or command returns a non-zero exit code or if there's any other error related to starting the temporary container. When the hook fails it will mark the build as failed.
+====
```

Comment:
this didn't work. i know what you want but aesthetically it looks bad. the ==== block is more for defining attributes rather than examples, otherwise it stretches till the bottom of the postcommit definition and looks like nothing similar in these docs.